packages/flutter_tools/lib/src/build_configuration.dart

```diff
@@ -81,37 +79,3 @@ HostPlatform getCurrentHostPlatform() {
 
   return HostPlatform.linux_x64;
 }
-
-TargetPlatform getCurrentHostPlatformAsTarget() {
-  if (Platform.isMacOS)
-    return TargetPlatform.darwin_x64;
-  if (Platform.isLinux)
-    return TargetPlatform.linux_x64;
-  printError('Unsupported host platform, defaulting to Linux');
-  return TargetPlatform.linux_x64;
-}
-
-class BuildConfiguration {
```

Comment:
We should rename this file from `build_configuration.dart` to something else - `builds.dart`? `build_info.dart`?